qcodes/instrument_drivers/QDevil/QDevil_QDAC.py

```diff
@@ -383,7 +383,7 @@ def _load_state(self) -> None:
 
                 self._assigned_fgs[chan] = Generator(fg)
                 self._assigned_fgs[chan].t_end = time_end
-                if trigger != 0:
```

Comment:
@jpsecher This looks to me like a bug in the (Qdevil QDac1) driver. On this line trigger is a string but its compared against 0 so that comparison will always fail. Does this check look ok to you

Reply:
yes, indeed! i think it's a very good catch!

Reply:
Yes, it seems like a bug